src/App.js

```diff
@@ -74,70 +76,76 @@ class App extends React.Component {
 
   componentDidMount() {
     API.getAllProducts()
-      .then(r => {
+      .then((r) => {
```

Comment:
What do the extra brackets do?

Reply:
ahh it's my editors lint rule has picked up that, with format on save, I've switched auto-save off now, as it causes a right pain at code review time, but this has probably been tainted now

https://eslint.org/docs/latest/rules/arrow-parens

basically enforces wrapping a single param in parens for clarity over brevity